virtcontainers/container.go

```diff
@@ -431,10 +431,10 @@ func (c *Container) createContainersDirs() error {
 	return nil
 }
 
-func (c *Container) shareFiles(m Mount, idx int, hostSharedDir, guestSharedDir string) (string, error) {
+func (c *Container) shareFiles(m Mount, idx int, hostSharedDir, guestSharedDir string) (string, bool, error) {
```

Comment:
@sboeuf it would be good to document what the return values actually mean or give them expressive names.